src/lib.rs

```diff
@@ -17,7 +17,6 @@ extern crate bincode;
 extern crate dav1d_sys;
 
 #[cfg(test)]
-#[macro_use]
```

Comment:
This is used by features decode_test and decode_test_dav1d.

Reply:
I forgot to add the `use interpolate_name::interpolate_test` indeed, now fixed :)